tests-common/src/main/kotlin/com/epam/drill/test/common/Util.kt

```diff
@@ -16,8 +16,8 @@
 package com.epam.drill.test.common
 
 import com.epam.drill.plugins.test2code.api.*
-import com.epam.drill.test.agent.TestListener.classParamsKey
-import com.epam.drill.test.agent.TestListener.methodParamsKey
+import com.epam.drill.test.agent.testinfo.CLASS_PARAMS_KEY
```

Comment:
Why do we even have this Util class and not implement extensions next to the types themselves

Reply:
A: Legacy